docs/data-sources/azure_cloud_service_deployment_targets.md

```diff
@@ -98,6 +98,7 @@ Read-Only:
 - `client_certificate_variable` (String)
 - `cloud_service_name` (String)
 - `cluster_certificate` (String)
+- `cluster_certificate_path` (String)
```

Comment:
Most changes in this PR are from a re-run of `make docs`. Looks like previous several merged PRs did not include the documentation updates. So these are picked up in this PR.